docs/user-guide/catalog.md

```diff
@@ -92,41 +96,117 @@ Enabling the *Show metadata template* option appears a text editor through witch
+* Select from a list known TMS services, with all the variants.
+
+<img src="../img/catalog/tms.jpg" class="ms-docimage"  style="max-width:500px;"/>
+<p class="ms-doc-caption">Select provider for TMS. The list of providers contains "custom", "TMS 1.0.0" and other resources</p>
+
+!!! note
+   Because some of these services are not standard, using them in different CRS may cause problems. So please take into account that switching CRS may cause problem when these layers are on the map.
+
+#### Custom TMS
+
+Selecting the **custom** provider the user can insert the tile URL template manually. The URL template is an URL with some placeholder that will be replaced with variables. The placeholder are identified by strings between brackets. e.g.: `{variable_name}`.
+
+<img src="../img/catalog/custom_tms.jpg" class="ms-docimage"  style="max-width:500px;"/>
+<p class="ms-doc-caption">Edit a custom TMS</p>
+
+
+Allowed placeholder are:
+
+* `{x}`,`{y}`,`{z}`: coordinates of the tiles
+* `{s}`: subdomains, this provides support for *domain sharding*. By default this is `["a", "b", "c"]`. User can customize the default by adding options.subdomains.
+
+*example:*
+
+```json
+{
+  "options": {
+    "subdomains": ["a", "b", "c", "d", "e"]
+  }
+}
+```
+
+When the user saves this custom catalog service and clicking on search, he will see only one result, that can be added to the map. (In the future custom provider can provide also variants, where in advanced settings the user can customize some variables, to have multiple results for the given resource).
+
+<img src="../img/catalog/custom_tms_browse.jpg" class="ms-docimage"  style="max-width:400px;"/>
+<p class="ms-doc-caption">Browse custom TMS service. It contains only one result</p>
+
+#### TMS 1.0.0
+
+Selecting the "TMS 1.0.0" provider the user can insert the URL of the Tile Map Service (see [TMS Specification](https://wiki.osgeo.org/wiki/Tile_Map_Service_Specification)). For instance, in GeoServer, it is the URL of the "TMS" link in the home page.
+
+<img src="../img/catalog/gs-tms.jpg" class="ms-docimage"  style="max-width:500px;"/>
+<p class="ms-doc-caption">TMS 1.1.0 URL from GeoServer</p>
+
+When saved this, the user will be allowed to browse and add to the map the TMS layers provided by the service. MapStore will filter the layers published showing only the tile maps in the current EPSG.
+
+<img src="../img/catalog/tms100_edit.jpg" class="ms-docimage"  style="max-width:400px;"/>
+<p class="ms-doc-caption">Edit a TMS 1.0.0 provider</p>
+
+<img src="../img/catalog/tms100_browse.jpg" class="ms-docimage"  style="max-width:400px;"/>
+<p class="ms-doc-caption">Browse TMS 1.0.0 layers</p>
+
+#### TMS Known Services
+
+The other known services are listed as providers below "custom" and "TMS 1.0.0". They are a static list configured inside the application. Selecting one of the provider listed and saving the new catalog service allows to browse al the variants known for that service.
```

Comment:
Please link the DEV guide section where the configuration of the static list is explained